### ISSUE TITLE
Trying to run tests - NPM task not found "lint:py", format:test"

### ISSUE THREAD

**Issue by sleighsoft** [plotly/dash-core-components#849]
I am sorry if this is a dumb question, but I am fairly new to npm. I followed the readme and when trying to run integration tests I get the following error.

```
npm run test
Debugger listening on ws://127.0.0.1:54725/48897a65-fd74-45a3-b5c2-f41de334dbbb
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.

> dash-core-components@1.10.2 test /home/dev/dash_playground/dash-core-components
> run-s -c lint lint:py format:test test-unit test:legacy test:intg test:pyimport

Debugger listening on ws://127.0.0.1:54733/bfdbe2d4-efe4-4590-8131-fcde224e66f7
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
ERROR: Task not found: "lint:py", format:test"
Waiting for the debugger to disconnect...
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dash-core-components@1.10.2 test: `run-s -c lint lint:py format:test test-unit test:legacy test:intg test:pyimport`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the dash-core-components@1.10.2 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
Debugger listening on ws://127.0.0.1:54725/48897a65-fd74-45a3-b5c2-f41de334dbbb
For help, see: https://nodejs.org/en/docs/inspector

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/dev/.npm/_logs/2020-08-22T07_54_40_992Z-debug.log
```

**Comment by sleighsoft** [comment]
Could you provide an example of how you run tests?

**Comment by alexcjohnson** [comment]
There's some info here but it's a bit buried: https://dash.plotly.com/testing

From the root of this repo with deps installed as in the readme, you can just type the command `pytest` to run all the Python tests, or use `pytest -k ...` to narrow your scope. We've set up the tests with names like `test_ddcf001_clearable_false_single` where `ddcf` identifies the file (dropdown clearable false) and `001` is the test number - so `pytest -k ddcf` will run all the tests in this file, and `pytest -k ddcf001` will just run that single test.

Aside from that there's `npm run lint` which just tests code formatting and conventions, and `npm run test-unit` which just collects a few JavaScript unit tests we have for individual components.

Closing this issue, now that you fixed it in #852 - thanks again!
